### DISCUSSION TITLE
Recover the crazyflies address

### DISCUSSION THREAD

**Discussion by Rishi23456** [bitcraze/discussions#1581] [Q&A]
Hello guys Im Vishanth. I am working on the swarm of drones, I have changed the address of 5 drones according to my requirement, out of the 5 drones 3 addresses got updated and getting connected while the other 2 drones address got misplaced, the crazyradio is not able to scan/connect to the 2 drones neither with the new address nor with the old address are there any steps using which the crazyflie can connect to the client without knowing address.

Thanks

**Comment by knmcguire** [comment]
Hi!

Best way is to connect a crazyflie through microUSB to your computer and press scan, and it will appear as usb0 in the cfclient. You can check and change the address from there. 

Just make sure that you have setup the usb rights (depended on your OS), such that the cfclient can access the crazyflie. The instructions can be found in the cfclient installation instructions https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/installation/install/

**Reply by Rishi23456** [reply]
I used Zadig for windows and it worked, Thank you
